src/cmd/flux-queue.py

```diff
@@ -143,12 +143,44 @@ def validate(self, path, conf):
                 raise ValueError(f"{path}: invalid key {key}")
 
 
+class QueueLimitsEffectiveRange:
+    """
+    QueueLimits wrapper which supports a effective range (configured minimum
+    to effective maximum minimum of limit or all resources in queue).
```

Comment:
is this sentence right?  "maximum minimum"?  Should it be "to effective maximum of all resources in queue"?

Reply:
oh wait I get it now ... maybe "to effective maximum, which is the minimum of the limit or all resources in the queue"

Reply:
D'oh! I merged this one after @cmoussa1's review. Can post a separate PR to fix these typos. (Or feel free to do it @chu11!) Thanks!